src/cobra/sampling/sampling.py

```diff
@@ -1,51 +1,58 @@
-# -*- coding: utf-8 -*-
+"""Provide a wrapper function for performing flux sampling of cobra models."""
 
-"""Module implementing flux sampling for cobra models."""
+from typing import TYPE_CHECKING, Optional
 
-from __future__ import absolute_import, division
+import pandas as pd
 
-import pandas
+from .achr import ACHRSampler
+from .optgp import OptGPSampler
 
-from cobra.sampling.achr import ACHRSampler
-from cobra.sampling.optgp import OptGPSampler
 
+if TYPE_CHECKING:
+    from cobra import Model
 
-def sample(model, n, method="optgp", thinning=100, processes=1, seed=None):
+
+def sample(
+    model: "Model",
+    n: int,
+    method: str = "optgp",
+    thinning: int = 100,
+    processes: int = 1,
+    seed: Optional[int] = None,
+) -> pd.DataFrame:
     """Sample valid flux distributions from a cobra model.
 
-    The function samples valid flux distributions from a cobra model.
-    Currently we support two methods:
+    Currently, two methods are supported:
 
     1. 'optgp' (default) which uses the OptGPSampler that supports parallel
-        sampling [1]_. Requires large numbers of samples to be performant
+        sampling. Requires large numbers of samples to be performant
```

Comment:
I find this sentence contradictory. `n < 1000` is a small sample size to me. So is the recommendation to run OptGP when n > 1000 or to run achr when n < 1000?

Reply:
I have missed to fix this information. If you look at `OptGPSampler` docs, it is recommended to use when `n > 1000`. I'll update it. Thanks!